### PR TITLE
remove ch1 placeholder and associated variables, functions, and files

### LINKED ISSUE #13: Chapter 1 placeholder map needs to be removed
* Chapter 1 placeholder map should be removed, so that the updated map can be inserted
* Requires updating html and css code in Set2.vue
* Requires removing ch1-specific variables and functions in Set2.vue
* Requires removing `states`, `segs_small`, and `states_merged` data layers in Set2.vue
* Requires deletion of associated data files from `data` folder

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
Addresses Issue #13 

Removes ch 1 placeholder map by:
* removing variables and functions associated with ch1 placeholder map
* updated html and css code to remove ch1 placeholder components
* removing code to load data files only used in ch1 placeholder map
* deleting data files only used in ch1 placeholder map

### CODE REVIEW

File: src/views/set2/Set2.vue [hunk 1]
```diff
@@ -365,10 +273,7 @@
             self.d3.json(self.publicPath + "data/NHDWaterbody_DelawareBay_pt6per_smooth.json"),
             self.d3.json(self.publicPath + "data/reservoirs.json"),
             self.d3.json(self.publicPath + "data/dams.json"),
```
Comment (jordansread) [inline src/views/set2/Set2.vue:275]: Couldn't find where `public/data/DRB_Wshed_1per_smooth.json` was referenced (is a deleted file). I assume it wasn't in the code and deleting that file was part of clean up?

Comment (jordansread) [reply src/views/set2/Set2.vue:275]: Likewise `Segments_subset_4per_smooth_pt5miBuffer_symdiff.json`, `matrix_obs.csv`, `narrative_rectangle.json`, and `observed_reach_station_coords.json`

**Review by jordansread — APPROVED** [review]
Looks good to me. Mostly clean up to prep for a new chapter 1.

File: src/views/set2/Set2.vue (hunk 1 above)
Comment (hcorson-dosch-usgs) [reply src/views/set2/Set2.vue:275]: Yes - Sorry, I should have been more clear. While I was going through and deleting the files associated with the ch1 stand-in map I realized there were some data files that I had used in earlier drafts but were not currently being used, so I deleted those as well.

**Pull request merged**